tests/conftest.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+import os
+import matlab.engine
+import shutil
+
+@pytest.fixture()
```

Comment:
Perhaps the scope of the fixture needs to be "module".

Reply:
Not necessary